product_state/readme/DESCRIPTION.rst

```diff
@@ -1 +1,6 @@
-This module introduces the state field on product template and allows simple product life cycle.
+This module introduces the state field on product template and allows simple product life cycle :
```

Comment:
Why putting an space between last word and `:`?

Reply:
Typographic habit. Removed.